### PR TITLE
Update get-started index page

### LINKED ISSUE #739: update get-started index page
https://dvc.org/doc - it mentions tutorials that have been moved to another section and a deep dive tutorial - I would replace it with a link to the Tutorials section and fix the text around

### PR DESCRIPTION
**Pull request by varadpoddar**
Fixed the link for " Tutorial " to link to the base tutorials directory and changed the surrounding text to a more readable and flowing paragraph.

### CODE REVIEW

File: static/docs/get-started/index.md [hunk 1]
```diff
@@ -17,6 +17,6 @@ us a ⭐ if you like the project!
 ✅ Contribute either [on GitHub](https://github.com/iterative/dvc) or
 [on Patreon](https://www.patreon.com/DVCorg/overview) to support the project.
 
-Separate to this section, the longer [Tutorial](/doc/tutorials/deep) also
-introduces DVC step-by-step, while additionally explaining in great detail the
-motivation and what's happening internally.
+The longer [Tutorials](/doc/tutorials) section contains a step-by-step introduction to DVC.
+In addition the section explains, in great detail, the motivation behind the project and its
+internal mechanism.
```
Comment (jorgeorpinel) [inline static/docs/get-started/index.md]: I like the better wording in this article but by "longer Tutorial" we specifically meant `[deep tutorial](/doc/tutorials/deep)` here. Please use that link instead 🙂

Comment (jorgeorpinel) [reply static/docs/get-started/index.md]: Maybe we should actually use `[Deep Dive Tutorial](/doc/tutorials/deep)` AND update the title to **Deep Dive Tutorial** in /doc/tutorials/deep (which currently only reads "**Tutorial**".

Comment (jorgeorpinel) [reply static/docs/get-started/index.md]: I just realized #739 states to change this link. OK, never mind! (I'll address the doc title separately.)

**Comment by varadpoddar** [comment]
@shcheklein 
Would you like me to continue re-wording the text to match your current structure?
😄

**Comment by shcheklein** [comment]
@varadpoddar yes if you are willing to try, it would be great :) without those additional changes it would be hard to merge this and it doesn't solve the issue.

**Comment by shcheklein** [comment]
@varadpoddar closing as inactive. Please, feel free to reopen and continue, the issue is still not solved, thanks!

**Pull request closed without merging**